packages/@aws-cdk/aws-iot-actions/test/cloudwatch-metric/integ.cloudwatch-metric-action.ts

```diff
@@ -0,0 +1,27 @@
+/// !cdk-integ pragma:ignore-assets
+import * as iot from '@aws-cdk/aws-iot';
+import * as cdk from '@aws-cdk/core';
+import * as actions from '../../lib';
+
```

Comment:
Put all of these files in the `test/cloudwatch` directory (now that https://github.com/aws/aws-cdk/pull/17495 has been merged).